app/views/RoomView/index.tsx

```diff
@@ -546,7 +592,8 @@ class RoomView extends React.Component<IRoomViewProps, IRoomViewState> {
 					room,
 					member,
 					showCloseModal: !!screen,
-					joined
+					joined,
+					omnichannelPermissions: [canForwardGuest, canReturnQueue, canViewCannedResponse]
```

Comment:
```suggestion
					omnichannelPermissions: {canForwardGuest, canReturnQueue, canViewCannedResponse}
```
As in the comment above.